docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,72 @@
+# Percona Everest: Frequently asked questions (FAQ)
+
+This section outlines the most frequently asked questions (FAQs) about Percona Everest.
+
+
+## What is a DatabaseEngine?
+
+The `DatabaseEngine` is a resource that represents the installed engines (MySQL, MongoDB, PostgreSQL). 
+
+Each `DatabaseEngine` indicates:
+
+- The operator version currently installed.
+- Compatible engine versions supported by that operator. 
+
+Refer to our [API documentation](https://percona-everest.readme.io/reference/getkubernetesclusterresources-1){:target="_blank"} for usage information.
+
+## Does Percona Everest provide logs for API calls?
+
+Yes, the Percona Everest backend has logs. It runs as an `everest-server` deployment within the `everest-system` namespace.
+
+- Accessing these logs helps us verify whether an API call was initiated for a specific user operation.
+
+- Identify any errors that may have occurred during that process.
+
+!!! note
+    Percona Everest logs are essential for troubleshooting issues when an operation is **completed**, but the corresponding Everest operator resource **fails to be created**.
+
+
+You can retrieve the logs from the pods associated with this deployment.
+
+To access them, run:
+
+```sh
+kubectl logs -f deploy/percona-everest -n everest-system
+```
+
+
+## How to troubleshoot issues between the UI and the Percona Everest API? 
+
+You can view the logs by running the following command:
+
+```sh
+kubectl logs -f deploy/percona-everest -n everest-system
+```
+
+## How to identify which component is failing?
+
+All communication with Percona Everest resources begins with the API.
+
+The API is responsible for updating Everest resources, while the Everest operator continues to create resources for the corresponding database operators. 
+
+### Recommended troubleshooting flow
```

Comment:
A sub-item isn't necessary if it's only one sentence.

Reply:
Fixed